test/integration/node-specific/operation_examples.test.ts

```diff
@@ -1785,13 +1779,13 @@ describe('Operation (Promises)', function () {
             o
           )
           .then(function (result) {
-            test.ok(result);
+            expect(result).to.exist;
```

Comment:
I can't seem to comment on the exact line but line 1777 is failing lint due to the empty function.

Reply:
done